.github/workflows/main.yaml

```diff
@@ -82,7 +82,7 @@ jobs:
     if: github.ref == 'refs/heads/main' || github.ref == 'refs/heads/develop'
     strategy:
       matrix:
-        os: ['ubuntu-20.04', 'ubuntu-22.04']
+        os: ['ubuntu-20.04', 'ubuntu-22.04', 'amazon_linux-2']
```

Comment:
The installer uses `amazon-linux` for naming. Using `amaon_linux` here so that the `awk` script below can still figure out the dist and ver. The underscore will be replaced with dash later one